inc/class-core-sitemaps-provider.php

```diff
@@ -308,6 +316,31 @@ function( $a, $b ) {
 		update_option( "core_sitemaps_lasmod_$suffix", $times[0] );
 	}
 
+	/**
+	 * Schedules asynchronous tasks to update lastmod entries for all sitemap pages.
+	 *
+	 * @return void
+	 */
+	public function update_lastmod_values() {
+		$sitemap_types = $this->get_object_sub_types();
+
+		foreach ( $sitemap_types as $type ) {
+			// Handle object names as strings.
+			$name = $type;
+
+			// Handle lists of post-objects.
+			if ( isset( $type->name ) ) {
+				$name = $type->name;
+			}
+
+			$total = $this->max_num_pages( $name );
+
+			for ( $page = 1; $page <= $total; $page ++ ) {
```

Comment:
FYI (outside the scope of this PR) There is protential here to reduce code duplication by passing in a callback into an abstracted function that calls the callback for each page, as this code is identical to `get_sitemap_entries()`.

Reply:
Yeah, I thought of that as well.